packages/node-utils/test/node-to-edge/request.test.ts

```diff
@@ -130,7 +130,7 @@ it('does not allow to read the body twice', async () => {
 
   expect(request.method).toEqual('POST')
   expect(await request.text()).toEqual('Hello World')
-  await expect(request.text()).rejects.toThrowError('The body has already been consumed.')
+  await expect(request.text()).rejects.toThrowError('Body is unusable')
```

Comment:
Does this new error message come from undici? The previous one looks more user-friendly to me.

Reply:
Yeah, this is from undici afaik. I can maybe map it but I'm not sure I want to <picture data-single-emoji=":sherthonk_holmes:" title=":sherthonk_holmes:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/sherthonk_holmes/90a6c2a6d6d4476a.png" alt=":sherthonk_holmes:" align="absmiddle"></picture> wdyt?